src/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 The Amber Authors.
+# Copyright 2018-2023 The Amber Authors.
```

Comment:
Please leave the copyright dates as they were.

Reply:
Done